src/routes/home/HomeView.vue

```diff
@@ -154,11 +154,6 @@ import HowItWorksDialog from '@/routes/home/components/HowItWorksDialog.vue';
 import GameStatus from '_/utils/GameStatus.json';
 import AnnouncementDialog from './components/announcementDialog/AnnouncementDialog.vue';
 
-const TABS = {
-  PLAY: 'play',
-  SPECTATE: 'spectate',
-};
```

Comment:
I'd prefer to keep this enum for clarity that these are the only expected tab values. We can revise their values to match the values you're using currently like
```
const tabs = {
  PLAY: '/',
  SPECTATE: '/spectate-list'
};
```